### PR TITLE
Expose all cassandra parameters. #282

### PR DESCRIPTION
**Pull request by zhiyanshao**
This is to address issue #282

### CODE REVIEW

File: universe/marathon.json.mustache [hunk 1]
```diff
@@ -146,6 +146,41 @@
...
 ,"CASSANDRA_COMMITLOG_TOTAL_SPACE_IN_MB":"{{cassandra.commitlog_total_space_in_mb}}"
+,"CASSANDRA_ROLES_UPDATE_INTERVAL_IN_MS":"{{cassandra.roles_update_interval_in_ms}}"
+,"CASSANDRA_PERMISSIONS_UPDATE_INTERVAL_IN_MS":"{{cassandra.permissions_update_interval_in_ms}}"
+,"CASSANDRA_KEY_CACHE_KEYS_TO_SAVE":"{{cassandra.key_cache_keys_to_save}}"
+,"CASSANDRA_ROW_CACHE_KEYS_TO_SAVE":"{{cassandra.row_cache_keys_to_save}}"
+,"CASSANDRA_COUNTER_CACHE_KEYS_TO_SAVE":"{{cassandra.counter_cache_keys_to_save}}"
+,"CASSANDRA_FILE_CACHE_SIZE_IN_MB":"{{cassandra.file_cache_size_in_mb}}"
+,"CASSANDRA_MEMTABLE_HEAP_SPACE_IN_MB":"{{cassandra.memtable_heap_space_in_mb}}"
+,"CASSANDRA_MEMTABLE_OFFHEAP_SPACE_IN_MB":"{{cassandra.memtable_offheap_space_in_mb}}"
+,"CASSAND,"RA_MEMTABLE_CLEANUP_THRESHOLD":"{{cassandra.memtable_cleanup_threshold}}"
```
Comment (verma7) [inline universe/marathon.json.mustache]: @zhiyanshao I don't think we should be modifying this file at all. The agreement with @triclambert and @gabrielhartmann was that it was fine to expose them as env variables that we at Uber can override, but Mesosphere does not want to expose all these parameters in the UI and complicate the configuration for DC/OS users.

Comment (zhiyanshao) [inline universe/marathon.json.mustache]: ok, I am going to revert the changes of this file and Mesosphere folks can update this file if they are needed.

**Comment by gabrielhartmann** [comment]
Not sure that's necessary. It's just the quotes not being closed properly.
But, as you like.

**Comment by gabrielhartmann** [comment]
@mesosphere-ci: retest this please

**Comment by gabrielhartmann** [comment]
@mesosphere-ci: retest this please

**Comment by gabrielhartmann** [comment]
@zhiyanshao:  Do you expect this PR to pass now?  Was hitting: 

```
Exception (org.apache.cassandra.exceptions.ConfigurationException) encountered during startup: Encryption must be enabled in client_encryption_options for native_transport_port_ssl
ERROR 00:53:17 Exception encountered during startup: Encryption must be enabled in client_encryption_options for native_transport_port_ssl
```
